docs/contributor-guide/developer-guide/technical-contribution-guide.md

```diff
@@ -18,9 +18,58 @@ Development tasks are managed in the [GitHub issues](https://github.com/civiform
 
 If you're just getting started, check out issues labeled with [Good First Issue](https://github.com/civiform/civiform/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). Also check out issues in the [next milestone](https://github.com/civiform/civiform/milestones?direction=asc\&sort=due\_date\&state=open) so you can work on the highest-priority tasks.
 
+### Git workflow
+
+Get the latest state from the github repository and create a new local development branch:
+
+```sh
+git fetch origin
+git checkout -b your_username/dev-branch-description
+git reset origin/main
+```
+
+Then start working on your change.
+
+When you're ready to submit your code for review, package your changes into a commit:
+
+```sh
+git status # make sure you are only adding the expected files.
+git add .
+git commit --message 'My change'
+```
+
+Push your local branch to the github repository:
+
+```sh
+git push --set-upstream origin your_username/dev-branch-description
+```
+
+If you need to make any changes before merging into origin/main, first get any new commits from the github repository and merge them into your local branch:
+
+```sh
+git fetch origin
```

Comment:
You don't really need to do this unless there are conficts, or it's been a while since you branched. 

But what you do need here is a `git pull` to incorporate any changes made from the UI or by other people to your branch

Reply:
I believe a fetch+merge is the same as a pull.  Updated to use pull.